src/Tasks/Microsoft.NET.Build.Tasks/SingleProjectInfo.cs

```diff
@@ -44,11 +51,12 @@ public static SingleProjectInfo Create(string projectPath, string name, string f
             }
 
             string outputName = name + fileExtension;
-            return new SingleProjectInfo(projectPath, name, version, outputName, resourceAssemblies);
+            return new SingleProjectInfo(projectPath, name, version, outputName, null, resourceAssemblies);
```

Comment:
nit: use named arguments for null literals

Reply:
Fixed.